raiden/tests/utils/blockchain.py

```diff
@@ -266,7 +266,7 @@ def geth_create_blockchain(
         nodedir = os.path.join(base_datadir, nodekey_part)
         node_genesis_path = os.path.join(nodedir, 'custom_genesis.json')
 
-        assert len(nodedir + '/geth.ipc') < 108, 'geth data path is too large'
+        assert len(nodedir + '/geth.ipc') <= 104, 'geth data path is too large'
```

Comment:
please, update the comment above

Reply:
The comment should be updated anyway. The reason isn't a go but rather an OS limit. 
On BSD (and therefore macOS) the socket path length limit is 104 chars.